### ISSUE TITLE
The result is overwriting previous result

### ISSUE THREAD

**Issue by aaksu13** [bahmutov/cypress-json-results#31]
I want to report in json format using the cypress-json-results plugin, but results are overwritten the json file for every run command sent from the local. How can I prevent this? Is it possible to open a new section in the json file for the result of each run command ?

**Comment by bahmutov** [comment]
I am sorry, I don't understand. What do you mean by "every run command sent from the local"? Are you doing something like
```
npx cypress run
// writes json results
npx cypress run
// overwrites json results
```

or do you mean that json results from one spec overwrite json results from previous spec?

**Comment by bahmutov** [comment]
I don't think this is true. Look at the CI for this repo, for example https://github.com/bahmutov/cypress-json-results/actions/runs/6341068720/job/17223878339 It shows two specs results in the same results JSON file

<img width="588" alt="Screenshot 2023-09-29 at 08 50 33" src="https://github.com/bahmutov/cypress-json-results/assets/2212006/a66710d1-d2e1-48f8-b5bd-d2b1a4b99cc5">
